codegen/src/sbt-test/generator/e2e/src/main/protobuf/demo.proto

```diff
@@ -23,10 +23,10 @@ message BarResp {}
 service Some {
 
   // foo does things with a FooReq
-  rpc Foo(FooReq) returns (FooResp) {}
+  rpc Foo(FooReq) returns (FooResp) {}  // foo does things with a FooReq
```

Comment:
why are the comments duplicated?

Reply:
I was trying to play with the "leading" and "trailing" comments, but it only seems to print the leading ones. I'm not sure why, the scalapb code seems like it should do both